azure-resources/DesktopVirtualization/hostPools/recommendations.yaml

```diff
@@ -1,102 +1,79 @@
   recommendationControl: Governance
-  recommendationImpact: Low
+  recommendationImpact: Medium
   recommendationResourceType: Microsoft.DesktopVirtualization/hostPools
   recommendationMetadataState: Active
   longDescription: |
-    At least one Validation Pool to have early warning if a planned update to AVD causes an issue. Also check that the host pool has been used regularly to test planned updates.
+    At least one Validation Pool to have early warning if a planned update to AVD causes an issue. support to adjust limits based on your business requirements. To handle a large number of users, consider scaling horizontally by creating multiple host pools.
+    Also check that the host pool has been used regularly to test planned updates.
     Host pools are a collection of one or more identical virtual machines within Azure Virtual Desktop environment. We highly recommend you create a validation host pool where service updates are applied first. Validation host pools let you monitor service updates before the service applies them to your standard or non-validation environment. Without a validation host pool, you may not discover changes that introduce errors, which could result in downtime for users in your standard environment.
     To ensure your apps work with the latest updates, the validation host pool should be as similar to host pools in your non-validation environment as possible. Users should connect as frequently to the validation host pool as they do to the standard host pool. If you have automated testing on your host pool, you should include automated testing on the validation host pool.
-  potentialBenefits: Early issue detection and testing for AVD updates
+  potentialBenefits: Enhanced environment stability
   pgVerified: true
   publishedToLearn: false
   publishedToAdvisor: false
   automationAvailable: no
   tags: null
   learnMoreLink:
     - name: Learn More
-      url: "https://learn.microsoft.com/en-us/azure/virtual-desktop/configure-validation-environment?tabs=azure-portal"
+      url: "https://learn.microsoft.com/azure/virtual-desktop/configure-validation-environment?tabs=azure-portal"
 
-- description: (Pooled) Configure scheduled agent updates
-  aprlGuid: 9fc522c1-d5b0-4bad-8169-1e1d32855afd
+- description: Configure host pool scheduled agent updates
+  aprlGuid: 979ff8be-5f3a-4d8e-9aa3-407ecdd6d6f7
   recommendationTypeId: null
-  recommendationControl: Scalability
+  recommendationControl: Governance
   recommendationImpact: Medium
   recommendationResourceType: Microsoft.DesktopVirtualization/hostPools
   recommendationMetadataState: Active
   longDescription: |
     Ensure schedules have been created to provide maintenance windows for AVD agent updates.
     The Scheduled Agent Updates feature lets you create up to two maintenance windows for the Azure Virtual Desktop agent, side-by-side stack, and Geneva Monitoring agent to get updated so that updates don't happen during peak business hours.
```

Comment:
```suggestion
    Create maintenance schedules for AVD agent updates to avoid disruptions. Use Scheduled Agent Updates to set maintenance windows for updating Azure Virtual Desktop agent, side-by-side stack, and Geneva Monitoring agent.
```